services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs_getters.py

```diff
@@ -360,24 +362,25 @@ async def get_job_pricing_unit(
 
 @router.get(
     "/{solver_key:path}/releases/{version}/jobs/{job_id:uuid}/logstream",
-    response_class=StreamingResponse,
+    response_class=LogStreamingResponse,
     include_in_schema=API_SERVER_DEV_FEATURES_ENABLED,
 )
 async def get_log_stream(
     solver_key: SolverKeyId,
     version: VersionStr,
     job_id: JobID,
-    log_listener: Annotated[LogListener, Depends(LogListener)],
     webserver_api: Annotated[AuthSession, Depends(get_webserver_session)],
+    director2_api: Annotated[DirectorV2Api, Depends(get_api_client(DirectorV2Api))],
+    log_distributor: Annotated[LogDistributor, Depends(get_log_distributor)],
+    user_id: Annotated[UserID, Depends(get_current_user_id)],
 ):
     job_name = _compose_job_resource_name(solver_key, version, job_id)
-    with log_context(_logger, logging.DEBUG, "Begin streaming logs"):
-        _logger.debug("job: %s", job_name)
+    with log_context(_logger, logging.DEBUG, f"Streaming logs for {job_name=}"):
```

Comment:
very minor: potentially ad the user_id as well since you have it

Reply:
Done